### DISCUSSION TITLE
Can you customize the Json serialization method?

### DISCUSSION THREAD

**Discussion by AbbadoLambily** [Netflix/dgs-framework#1025] [General]
Can you customize the Json serialization method?

**Comment by srinivasankavitha** [comment]
Can you provide a few more details on where you are looking to customize the json serialization? Is it during query execution for returning the response? If so, we do offer a way to set your own object mapper.: https://netflix.github.io/dgs/advanced/custom-object-mapper/

**Comment by srinivasankavitha** [comment]
Also, for questions, please open a discussion thread and we can convert it to an issue if needed.
